crates/uv-cli/src/lib.rs

```diff
@@ -186,9 +186,11 @@ pub struct GlobalArgs {
     #[arg(global = true, long, hide = true)]
     pub python_fetch: Option<PythonDownloads>,
 
-    /// Do not print any output.
-    #[arg(global = true, long, short, conflicts_with = "verbose")]
-    pub quiet: bool,
+    /// Use quiet output.
+    ///
+    /// `-q` will produce less stdout output while `-qq` will produce no stdout output.
```

Comment:
Should we avoid updating this for now and track once we start using it?

I would start with

```suggestion
    /// Repeating this option, e.g., `-qq`, will enable a silent mode in which uv will write no output to stdout.
```